node/tests/utils.rs

```diff
@@ -359,10 +359,17 @@ impl MASQNode {
         ensure_start: bool,
         command_getter: F,
     ) -> MASQNode {
-        let data_dir = if sterile_database {
-            ensure_node_home_directory_exists("integration", test_name)
-        } else {
-            node_home_directory("integration", test_name)
```

Comment:
If the `config_opt` contains a `--data-directory` specification, it shouldn't be ignored like this.